js/client/modules/@arangodb/aql-helper.js

```diff
@@ -300,7 +328,10 @@ function getQueryMultiplePlansAndExecutions (query, bindVars, testObject, debug)
     require('internal').print(require('js-yaml').safeDump(plan));
   };
 
-  const paramNone = { optimizer: { rules: [ '-all' ]},  verbosePlans: true};
+  const paramNone = { optimizer: { rules: [ '-all' ]},
+                      verbosePlans: true,
+                      batchSize: "none"
```

Comment:
That doesn't look correct, I think `batchSize` must be a positive integer. Why set it at all, and not use the default?